app/packs/src/stores/alt/stores/SpectraStore.js

```diff
@@ -80,9 +80,12 @@ class SpectraStore {
 
   handleLoadSpectra({ fetchedFiles, spcInfos }) {
     const spcMetas = this.decodeSpectra(fetchedFiles);
-    const newArrSpcIdx = spcMetas.map(spci => (
+    let newArrSpcIdx = spcMetas.map(spci => (
       spci.idx
     )).filter(r => r !== null);
+    if (newArrSpcIdx.length <= 1) {
```

Comment:
Just double checking: so we need at least two elements in our filtered array, otherwise we set it `[]`, right?
only one element is not enough?

Reply:
Yes, because chemspectra only supports the display of multiple spectra at the same time when the spectra files is a Cyclic Voltammetry spectra for now. So in the case of only on spectra file, we treat it like the others.